### DISCUSSION TITLE
ISO for Beta or Git pull install only?

### DISCUSSION THREAD

**Discussion by rsneeden** [Security-Onion-Solutions/securityonion#10045] [2.4] [answered]
Just curious if there will be an ISO install for beta or is it just the git pull install direct to Rocky/Ubuntu ?

-TheOGStacy (Stacy Sneeden)

**Comment by reyesj2** [comment, marked as answer]
#10031 

> There is no ISO for beta 1. You need to install a rocky linux MINIMAL install. The ISO is not quite ready yet due to how elastic agent is packaged making the ISO 11GB so we are working on getting that down.

**Reply by rsneeden** [reply]
Gotcha. I did an managersearch install last night on Rocky Minimal with some errors. I'll review and if it's relevant I'll post data later today.

I've found all previous 2.x versions really really really need you to have your IP, DNS server(s) and hostname strategy 100% and so my lab is virtual since I'm traveling and I just need to tighten that up I'm sure.

**Reply by reyesj2** [reply]
Thanks for testing out 2.4 beta 1!

Ensure you're using the Rocky9-minimal ISO https://download.rockylinux.org/pub/rocky/9/isos/x86_64/Rocky-9.1-x86_64-minimal.iso
But yes, since this is a network install you need to have the relevant network configurations done before running the so-setup-network installer

**Reply by rsneeden** [reply]
Yes, I've got the base OS and I'm also going to do some Ubuntu testing as well, even thought it's being phased out.

I'm rebuilding again this afternoon...
